CONTRIBUTING.md

```diff
@@ -0,0 +1,144 @@
+# Contributing to SKY UX
+
+We encourage contributions from all SKY UX users. To contribute, follow this step-by-step guide. For more information about our contribution guidelines, see [SKY UX Contribution Guidelines](https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines).
```

Comment:
```suggestion
We encourage contributions from all SKY UX users. To contribute, follow this step-by-step guide. For more additional guidance, see the [SKY UX Contribution Guidelines](https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines).
```
Disregard if you don't like these edits. I thought we should have "the" before the link in the last sentence and tweaked the opening clause to make it flow slightly better.